### DISCUSSION TITLE
Different aspect ratios when using multiple images

### DISCUSSION THREAD

**Discussion by yorkim** [ErikHen/PictureRenderer.Optimizely#23] [Ideas]
Hi,

I have a use case where I would like to have a single picture element where i render both portrait and landscape versions of an image depending on screen size. I can not see that this is possible with the current solution (please correct me if I'm wrong).

I assume this could be solved by adding an aspect ratio parameter to the `MediaCondition` class and then use that ratio for rendering. This might also apply to the focal point property since one might want different focal points when creating both portrait and landscape images from the same image.

**Comment by ErikHen** [comment]
Hi, you are correct that that isn't supported. But I guess you can do that by having two picture elements, and show/hide depending on viewport width (using css)?
